samples/CMakeLists.txt

```diff
@@ -1,6 +1,103 @@
 # Copyright (C) 2018-2025 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
+
+if (NOT GENAI_SAMPLES_ARCHIVE_OUTPUT_DIRECTORY)
+    if (CMAKE_ARCHIVE_OUTPUT_DIRECTORY)
+        set (GENAI_SAMPLES_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_ARCHIVE_OUTPUT_DIRECTORY}/samples/")
+    else()
+        set (GENAI_SAMPLES_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/samples/")
+    endif()
+endif()
+
+if (NOT GENAI_SAMPLES_LIBRARY_OUTPUT_DIRECTORY)
+    if (CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+        set (GENAI_SAMPLES_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/samples/")
+    else()
+        set (GENAI_SAMPLES_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/samples/")
+    endif()
+endif()
+
+if (NOT GENAI_SAMPLES_RUNTIME_OUTPUT_DIRECTORY)
+    if (CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+        set (GENAI_SAMPLES_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/samples/")
+    else()
+        set (GENAI_SAMPLES_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/samples/")
```

Comment:
These vars are already set in `cmake/global_properties.cmake`. Why set them again here?

Reply:
removed.